test/recipes/80-test_cmp_http.t

```diff
@@ -283,14 +281,22 @@ sub start_mock_server {
             print;
             s/\R$//;                # Better chomp
             next unless (/^ACCEPT/);
-            $server_port = $server_tls = $kur_port = $pbm_port = $1
-                if m/^ACCEPT\s.*?:(\d+)$/;
+
+            # $1 may be undefined, which is OK to assign to $server_port,
+            # as that gets detected further down.
+            /^ACCEPT\s.*:(\d+)$/;
+            $server_port = $1;
+
             last;
         }
+
+        unless (defined $server_port) {
```

Comment:
On this occasion I suggest also fixing an issue that was there already before and that I noticed today:
In case the server does not emit ACCEPT but terminates, `$server_port` will still be `"0"`.
So better add to the condition in this line: `&& $server_port != 0`.

Reply:
I make a change, not quite what you asked for, but should have the same effect.